libs/npress/models/PagesModel.php

```diff
@@ -684,7 +694,7 @@ public function getRedirectLink()
     return $pagelink;
   }
 
-  public function link($absolute = false)
+  public function link($id_version,$absolute = false )
```

Comment:
Tohle asi rozbije všechna další použití `->link(true)`. Doporučil bych dát id_version parametr až jako druhý a nastavit mu default `link($absolute=false, $id_version = "")`, protože se absolute asi používá častěji.